hack/quickstart/kubelet.master

```diff
@@ -32,7 +34,9 @@ ExecStart=/usr/lib/coreos/kubelet-wrapper \
   --network-plugin=cni \
   --node-labels=node-role.kubernetes.io/master \
   --pod-manifest-path=/etc/kubernetes/manifests \
-  --register-with-taints=node-role.kubernetes.io/master=:NoSchedule
+  --register-with-taints=node-role.kubernetes.io/master=:NoSchedule \
+  --require-kubeconfig \
```

Comment:
I think this is deprecated

Reply:
yup https://github.com/kubernetes/kubernetes/commit/2189314895a28e37f30781c721bddd9228001c29

Reply:
Thanks, will remove when I rebase for https://github.com/kubernetes-incubator/bootkube/pull/767